openapi/paths/payout-requests@{id}@payment-instruments.yaml

```
@@ -0,0 +1,58 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Transactions
+  summary: Retrieve associated payment instruments
+  operationId: GetPayoutRequestPaymentInstruments
+  x-sdk-operation-name: getPaymentInstruments
+  description: Retrieves the payment gateway and a list of payment instruments that are associated with a payout request with a specified ID.
+  responses:
+    200:
+      description: Payout request associated payment instruments retrieved.
+      content:
+        application/json:
+          schema:
+            type: array
+            description: |-
+              List of available payment instruments per an associated gateway to offer for the payout.
```

nitpick: we should avoid adding block scalars for one liners
```suggestion
            description: List of available payment instruments per an associated gateway to offer for the payout.
```